mods/ra/maps/soviet-02b/map.yaml

```diff
@@ -0,0 +1,681 @@
+MapFormat: 7
+
+RequiresMod: ra
+
+Title: Soviet 02b: The thin red line
+
+Description: Tomorrow, the attack on Germany begins, but today, we must protect our facility from Allied attacks.\n\nKeep the Command Center intact at all costs, and destroy any Allied fortification you might find.
+
+Author: Westwood Studios
+
+Tileset: TEMPERAT
+
+MapSize: 128,128
+
+Bounds: 36,52,54,35
+
+Visibility: MissionSelector
+
+Type: Campaign
+
+Videos:
+	Briefing: soviet2.vqa
+	GameStart: spotter.vqa
+	GameWon: sovtstar.vqa
+	GameLost: sovcemet.vqa
+
+Options:
+	Crates: False
+	Fog: True
+	Shroud: True
+	AllyBuildRadius: False
+	FragileAlliances: False
+	StartingCash: 5000
+	ConfigurableStartingUnits: False
+	ShortGame: False
+
+Players:
+	PlayerReference@Neutral:
+		Name: Neutral
+		OwnsWorld: True
+		NonCombatant: True
+		Faction: allies
+	PlayerReference@Creeps:
```

Comment:
This extra player is unnecessary. You can make the barrels that are currently owned by Creeps be owned by the soviets.